protocols/swap/test/Swap-unit.js

```diff
@@ -105,12 +105,12 @@ contract('Swap Unit Tests', async accounts => {
         signature,
       ]
 
-      //sender authorizes signer
+      //sender authorizes signer using authorizeSender
```

Comment:
This comment doesn't make sense. Is the comment wrong or the code wrong?

Reply:
think both are right, though have expanded on the party names

      //mockSender authorizes mockSigner using authorizeSender
      //mocks that case where one can send unsigned orders if authorized

Reply:
Right, I think what's confusing is that the sender is authorising the signer to send (when the name `signer` means you assume they are taking the role of signing).

Maybe a comment saying something along the lines of `// the roles of this authorization need to be reversed for the swap to succeed` ?

Reply:
We cleared up this misunderstanding face to face - I had explained myself very badly here. The code following these lines is redundant to the test. Deepa verified this and is going to remove these lines from the tests.

Reply:
line has been removed and tests stay at 100%